qa/automation/Alleva 2405/Sign_in_to_app_with_Google_account.feature

```diff
@@ -0,0 +1,15 @@
+Feature: 1.1. Sign in to App with Google account
+
+Scenario: Sign in with Google account granted
+	Given user is on https://qa.ddso-spot.quantori.com/ page
+	When user hits "Sign in with Google" button
+	And "Google account authentication" is granted by Google
+	Then user is signed in to the app
+	And user is taken to "Dataset" page of the app
+
+Scenario: Sign in with Google account is not granted
+	Given user is on https://qa.ddso-spot.quantori.com/ page
+	When user hits "Sign in with Google" button
+	And "Google account authentication" is not granted by Google
+	Then user sign in to the app is denied
+	And user is taken to "Sign in with Google" page
```

Comment:
About the whole PR: 
1) There was no need to delete previous version of your PR. Because it was great - coverage, ideas of test scenarios. Great job! I couldn't leave comments there because it's not a current version, unfortunately. Yes, it's not our "Sign in" page (Google), but your task was to test it and to think what user can do here, can he log in with invalid password or if he doesn't have Google account etc.
2) As I mentioned before - we should use concrete data (email, password etc)
3) Pay attention to syntax (especially in When and Then steps)